test/conformance/errorcondition_test.go

```diff
@@ -118,12 +118,6 @@ func TestContainerErrorMsg(t *testing.T) {
 // for the container crashing scenario.
 func TestContainerExitingMsg(t *testing.T) {
 	t.Parallel()
```

Comment:
I believe you can delete `t.Parallel()` here as it is called by each test later.

Reply:
Deleted.

Reply:
The original `t.Parallel()` does change the behavior of this test. A `t.Parallel()` in subtests will allow subtests to run in parallel, but will not allow the overall test to run parallel with other conformance tests.

See about halfway down https://blog.golang.org/subtests for details.

Reply:
Thanks for the reference. Added back the original `t.Parallel()` to allow this group of tests to be run in parallel along with the other conformance test.